source/common/network/listen_socket_impl.cc

```diff
@@ -17,6 +18,21 @@ void ListenSocketImpl::doBind() {
   }
 }
 
+TcpListenSocket::TcpListenSocket(const std::string& address, bool bind_to_port) {
```

Comment:
Matt raised similar issue above. In config, admin still uses port. So we can have a follow up PR to update admin from using port to address and make corresponding change later. I will let him decide